src/chat.rs

```diff
@@ -3461,8 +3461,8 @@ pub(crate) async fn add_contact_to_chat_ex(
     if chat.typ == Chattype::Group && chat.is_promoted() {
         msg.viewtype = Viewtype::Text;
 
-        let contact_addr = contact.get_addr();
-        msg.text = stock_str::msg_add_member_local(context, contact_addr, ContactId::SELF).await;
+        let contact_addr = contact.get_addr().to_lowercase();
+        msg.text = stock_str::msg_add_member_local(context, &contact_addr, ContactId::SELF).await;
```

Comment:
i thought first that it would be better to use the address as given in the member list, however, i agree, that case used by ppl is often not the address that is meant to be used. so +1 for lowercasing even here